### ISSUE TITLE
[iOS] Uses Asset Catalog for setting icon

### ISSUE THREAD

**Issue by dumganhar** [cocos2d/cocos2d-x#18306]
Could refer to :
https://developer.apple.com/ios/human-interface-guidelines/icons-and-images/app-icon/
https://developer.apple.com/library/content/qa/qa1686/_index.html

**Comment by minggo** [comment]
I don't think we should use asset catalog, as [discussed here](http://discuss.cocos2d-x.org/t/iphone-x-slow-transitions-xcode-bug-and-not-full-screen-solved/38942).
